src/components/grid-aware/StatsBlock/StatsBlock.module.css

```diff
@@ -92,6 +97,7 @@
   font-size: 72px;
   font-weight: bold;
   line-height: 1.3;
+  min-width: 100px;
```

Comment:
Is this still needed with the other two changes?

Reply:
Good catch. It's not needed. I will delete it.